src/screens/Drawer/NewAssignmentsScreen.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react';
+import { Layout, Text } from '@ui-kitten/components';
+import {
+    FlatList,
+    StyleSheet,
+    TouchableOpacity,
+    LayoutAnimation,
+    Platform,
+    UIManager,
+    View,
+} from 'react-native';
+import { Screen } from '../../components';
+import { useSelector } from 'react-redux';
+import { LinearGradient } from 'expo-linear-gradient';
+import { openDrawerAction } from '../../shared/components';
+import SwipeRow from '../../components/SwipeableItem';
+import { NewAssignmentItem } from '../../components/NewAssignments/NewAssignmentItem';
+import { windowWidth } from '../../shared/constants';
+
+interface NewAssignmentsScreenProps {
+    route;
+    navigation;
+    onSwipe;
+}
+
+if (Platform.OS === 'android') {
+    UIManager.setLayoutAnimationEnabledExperimental &&
+        UIManager.setLayoutAnimationEnabledExperimental(true);
+}
+
+export const NewAssignmentsScreen = (props: NewAssignmentsScreenProps) => {
+    //grab new assignments instead of tasks
+    const newAssignments = useSelector((state) => state.taskReducer.tasks);
```

Comment:
useEffect(() => {}, [])

Reply:
changed state to just take newAssignments instead of () => newAssignments